tests/mobly/controllers/android_device_lib/adb_test.py

```diff
@@ -628,6 +628,22 @@ def test_getprops_when_empty_string_always_returned(self, mock_sleep):
   def test_forward(self):
     with mock.patch.object(adb.AdbProxy, '_exec_cmd') as mock_exec_cmd:
       adb.AdbProxy().forward(MOCK_SHELL_COMMAND)
+      mock_exec_cmd.assert_called_with(['adb', 'forward', 'ls'],
+                                       shell=False,
+                                       timeout=None,
+                                       stderr=None)
+
+  def test_reverse(self):
+    with mock.patch.object(adb.AdbProxy, '_exec_cmd') as mock_exec_cmd:
+      adb.AdbProxy().reverse(MOCK_SHELL_COMMAND)
+      mock_exec_cmd.assert_called_with(['adb', 'reverse', 'ls'],
+                                       shell=False,
+                                       timeout=None,
+                                       stderr=None)
+
+  def test_reverse(self):
```

Comment:
why are there two `test_reverse`?

Reply:
Oh. I think i made some mistake while committing the code. A little rusty on git. I'll fix it.

Reply:
Fixed it.